common/component_interface_specs/test/test_control.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 the Autoware Foundation
```

Comment:
Since it is a new file, please change the publication year to 2023 and delete the Co-developed. Also files for other components.

Reply:
Thank you for the update. However, please make some changes to comply with guideline.
- https://github.com/autowarefoundation/autoware.universe/pull/5034#pullrequestreview-1645793682
- https://github.com/autowarefoundation/autoware.universe/pull/5034#pullrequestreview-1645836963